platform/debian/build_kanidm.sh

```diff
@@ -46,8 +46,8 @@ if [ "$(which cargo | wc -l)" -eq 0 ]; then
     fi
 fi
 
-# this assumes the versions are in lock-step, which is fine at the moment.s
-KANIDM_VERSION="$(grep -ioE 'version.*' kanidm_tools/Cargo.toml | head -n1 | awk '{print $NF}' | tr -d '"')"
+# this assumes the versions are in lock-step, which is fine at the moment.
```

Comment:
# Comments may be out of date 🤣

The root cargo.toml is the only one that sets a version with rust1.64 now :)

Reply:
🤣

Reply:
Sorry for the SCREAMING it was meant to be a '#'